include/El/core/Grid.hpp

```diff
@@ -132,6 +134,92 @@ inline void AssertSameGrids( const Grid& g1, const Grid& g2, Args&... args )
     AssertSameGrids( g2, args... );
 }
 
+inline bool GridCompare
```

Comment:
Is there a case where ``GridCompare`` is needed (rather than reusing the same ``Grid`` instance)? This routine seems fairly expensive (in addition to requiring another ``friend`` declaration). If it's to be kept, I think that checking that the relevant communicators are identical rather than just congruent could be faster and less code.